homeassistant/components/homekit_controller/sensor.py

```diff
@@ -166,7 +166,7 @@ def thread_status_to_str(char: Characteristic) -> str:
         key=CharacteristicsTypes.VENDOR_CONNECTSENSE_ENERGY_KW_HOUR,
         name="Energy kWh",
         device_class=SensorDeviceClass.ENERGY,
-        state_class=SensorStateClass.MEASUREMENT,
+        state_class=SensorStateClass.TOTAL_INCREASING,
```

Comment:
According to the PR description:
> The value is the total use, always increasing, so this change just updates the default sensor class to match the real value.

Should this be `SensorStateClass.TOTAL` then? `SensorStateClass.TOTAL_INCREASING` should be used only for sensors which periodically reset to 0: https://developers.home-assistant.io/docs/core/entity/sensor#how-to-choose-state_class-and-last_reset